packaging/packager_rpm.sh

```diff
@@ -5,40 +5,44 @@ readonly distro="${DISTRO:-fedora}"
 readonly distro_version="${VERSION:-30}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# rpm dependencies
-# python3
-dnf -y install python3 python3-coverage
-# python-imaging
+echo "------------- python3 -------------"
+dnf -y install python3 python3-coverage python3-devel
+echo "------------- python-imaging -------------"
 dnf -y install python3-pillow
-# pip dependencies (for dependencies not available as RPM)
+echo "-------------  pip dependencies (for dependencies not available as RPM) -------------"
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+echo "------------- upgrade pip  -------------"
 pip3 install --upgrade pip
-# contour, template, feature, cascade, text matching
+
+echo "------------- contour, template, feature, cascade, text matching -------------"
 dnf -y install python3-numpy python3-opencv
-# text matching
 dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
-pip3 install pytesseract==0.3.4 tesserocr==2.5.1
-# deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
-# screen controlling
-if (( distro_version <= 32 )); then
-    pip3 install autopy==4.0.0
+pip3 install pytesseract==0.3.13 tesserocr==2.7.1
+
+echo "------------- deep learning -------------"
+dnf -y install python3-torch python3-torch-devel
+pip3 install torchvision==0.17.0
+# TODO: python3-torchvision version available is broken (no module found)
+
+echo "------------- screen controlling -------------"
+echo "DISABLE_AUTOPY is set to: '$DISABLE_AUTOPY'"
+if [[ -n "$DISABLE_AUTOPY" && "$DISABLE_AUTOPY" == "1" ]]; then
```

Comment:
This ignores the previous logic where we will always install the autopy for fedora version <32.

Reply:
Also the unnecessary echo statements